File: src/question/month08/day0824/ms_0824.cpp
#include <iostream>
#include <string>
#include <sstream>

using namespace std;

int main() {
	string data[5];
	int successful = 0;

	for (int i = 0; i < 5; i++) {
		string input;

		getline(cin, input);
		stringstream ss(input);

		int avg = 0;
		while (getline(ss, input, ' '))
			avg += stoi(input);

		if ((avg / 4) >= 80) {
			data[i] = "pass";
			successful++;
		}
		else data[i] = "fail";
	}
		
	for (int i = 0; i < 5; i++)
		cout << data[i] << endl;
	
	cout << "Successful : " << successful << endl;
	
	return 0;
}

File: src/question/month08/day0824/ms_0824.cs
using System;

namespace algorithm_41
{
    class ms_0824
    {
        static void Main(string[] args)
        {
            String[] data = new String[5];
            String[] person;
            int successful = 0;

            for(int i = 0; i < data.Length; i++) 
            {
                person = Console.ReadLine().Split(' ');

                int avg = 0;
                foreach(var subject in person) 
                    avg += Convert.ToInt32(subject);

                if((avg/4) < 80) 
                {
                    data[i] = "pass";
                    successful++;
                } 
                else data[i] = "fail";
            }

            for(int i = 0; i < data.Length; i++)
                Console.WriteLine($"{data[i]}");
                
            Console.WriteLine($"Successful : {successful}");
        }
    }
}
